File: tests/expected/assert.go
package main

import (
	"fmt"
)

func CompareAssert(a int, b int) {
	if !(a == b) {
		panic("assert")
	}
	if !(!(0 == 1)) {
		panic("assert")
	}
}

func main() {
	if !(true) {
		panic("assert")
	}
	if !(!(false)) {
		panic("assert")
	}
	CompareAssert(1, 1)
	if !(true) {
		panic("assert")
	}
	if !(true) {
		panic("assert")
	}
	fmt.Printf("%v\n", "OK")
}

File: tests/expected/assert.jl
function compare_assert(a::Int64, b::Int64)
    @assert(a == b)
    @assert(!(0 == 1))
end

if abspath(PROGRAM_FILE) == @__FILE__
    @assert(true)
    @assert(!false)
    compare_assert(1, 1)
    @assert(true)
    @assert(true)
    println("OK")
end
